tests/e2e/provisioning/go.mod

```diff
@@ -1,78 +1,79 @@
 module github.com/kyma-project/control-plane/tests/e2e/provisioning
 
-go 1.19
+go 1.21
 
 require (
-	github.com/google/uuid v1.3.0
+	github.com/google/uuid v1.4.0
 	github.com/kyma-incubator/compass/tests/director v0.0.0-20210323111309-0e91422c2c76
-	github.com/kyma-project/control-plane/components/kyma-environment-broker v0.0.0-20230418152422-ee68639cb3ab
-	github.com/kyma-project/control-plane/components/provisioner v0.0.0-20230418152422-ee68639cb3ab
+	github.com/kyma-project/control-plane/components/provisioner v0.0.0-20231228092043-11c5ace49013
+	github.com/kyma-project/kyma-environment-broker v0.0.1
```

Comment:
Why v0.0.1?

Reply:
There is something wrong with our components. I am not sure why `go get` sees version 1.0.2 as incorrect and missing. I will check that later to see what is wrong but for now I would like to get the imports sorted out.

Reply:
It works like so:
```
➜  provisioner git:(features/keb-cleanup) ✗ go get -v  github.com/kyma-project/kyma-environment-broker@1.0.2
go: downloading github.com/kyma-project/kyma-environment-broker v0.0.0-20231227095843-e35b81fff501
```
There is a convention in Go to use `v` before the version. Let's move it to separate issue and I will update version correctly after it will have been discussed.

Reply:
The issue: https://github.com/kyma-project/kyma-environment-broker/issues/314